conf/Makefile.chibios

```diff
@@ -39,7 +60,6 @@ Q=@
 RTOS_DEBUG ?= 0
 
 ifeq (,$(findstring $(RTOS_DEBUG),0 FALSE))
-$(info DEBUGGING ON)
```

Comment:
I found it annoying - since most of the time I was compiling I had the debugging off. Maybe have it display the message only if the debugging is ON? And make it more descriptive - such as " ChibiOS debug mode is ON" - sounds good?